birdhouse/components/stac/docker-compose-extra.yml

```diff
@@ -0,0 +1,63 @@
+version: "3.4"
+
+x-logging:
+  &default-logging
+  driver: "json-file"
+  options:
+    max-size: "50m"
+    max-file: "10"
+
+services:
+  stac:
+    container_name: stac
+    image: ghcr.io/crim-ca/stac-app:main
```

Comment:
Not using exact version for reproductibility?

Reply:
https://github.com/crim-ca/stac-app/issues/1